docs/zh/guides/advanced/database/field/system-info/field-created-by.md

```
@@ -1 +1,11 @@
-# 创建人
\ No newline at end of file
+# 创建人
+
+### 介绍
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)